### PR TITLE
Text proposal txn by multisig

### PR DESCRIPTION
**Pull request by amandeeprehal-lab49**
Text proposal txn by multisig

### CODE REVIEW

File: contracts/dao/MultisigDAO.sol [hunk 1]
```diff
@@ -36,6 +36,7 @@ contract MultiSigDAO is BaseDAO {
 
     uint256 private constant TXN_TYPE_TOKEN_TRANSFER = 1;
     uint256 private constant TXN_TYPE_BATCH = 2;
+    uint256 private constant TXN_TYPE_TEXT = 3;
```
Comment (shivamgarglab49) [inline contracts/dao/MultisigDAO.sol:39]: this constant is not used.

File: integrationTest/dao/multiSigDAO.ts [hunk 2]
```diff
@@ -166,6 +168,58 @@ export async function executeDAO(
...
+  console.log(
+    `- executing text proposal using Multi-sig DAO  = ${multiSigDAO.contractId}\n`
+  );
+
+  const gnosisSafe = await getGnosisSafeInstance(multiSigDAO);
+
+  const textProposalText = "TEXT";
+
+  const textTxData = await multiSigDAO.encodeFunctionData(
+    "MultiSigDAO",
```
Comment (shivamgarglab49) [inline integrationTest/dao/multiSigDAO.ts:185]: please use const declared inside ContractServcice and also create one const for "setText" fun.

File: contracts/dao/MultisigDAO.sol (hunk 1 above)
Comment (amandeeprehal-lab49) [reply contracts/dao/MultisigDAO.sol]: done

File: integrationTest/dao/multiSigDAO.ts (hunk 2 above)
Comment (amandeeprehal-lab49) [reply integrationTest/dao/multiSigDAO.ts]: done

**Review by shivamgarglab49 — APPROVED** [review]

**Pull request merged**